app/views/juntos_bootstrap/shared/_header_big.html.slim

```diff
@@ -6,4 +6,10 @@
   = link_to t('layouts.header.submit'), start_path(submit: true), class: 'w-hidden-small w-hidden-tiny header-link w-nav-link'
 
   = link_to t('layouts.header.how_it_works'), start_path, class: 'header-link w-nav-link'
+
+
+  = link_to 'PT', @post, class: 'btn-translation'
+
+  = link_to 'EN', @post, class: 'btn-translation'
```

Comment:
O que tem no resultado de `I18n.available_languages`? Se o resultado for apenas PT e EN acho que talvez role um `each` aqui.